src/test/resources/datasets/user/changing_password_or_email/changing_password_or_email_expected.yml

```diff
@@ -0,0 +1,17 @@
+user:
+  - id: 5
+    username: 'trump'
```

Comment:
It's not a good idea. Once we get into production our code would be indexed by search systems. Politics and kids needs to be separated.